Makefile

```diff
@@ -29,8 +29,6 @@ build-linux-builders: build-builder-alpine build-builder-bionic
 build-builder-alpine: build-linux-packages build-sample-root
 	@echo "> Building 'alpine' builder..."
 	$(PACK_CMD) builder create cnbs/sample-builder:alpine --config $(SAMPLES_ROOT)/builders/alpine/builder.toml $(PULL_POLICY_NEVER)
-	docker run cnbs/sample-builder:alpine ls /cnb/extensions/samples_curl
```

Comment:
Ah well, I guess we could keep the checks if we want - they were added previously because I was pulling the sample builder and wondering, what could possibly be going on?

Thinking about this more, the alpine builder in its current state is actually broken 😞 for [demo purposes](https://github.com/buildpacks/docs/blob/main/content/docs/extension-author-guide/create-extension/why-dockerfiles.md#build-the-application-image) which is not ideal. I think I will simplify the tutorial (and make the builder working by default) by introducing a `BP_` environment variable to control the buildpack behavior WRT extensions.

TLDR; I'll update this PR and create a corresponding docs PR to make everything better

Reply:
Added the aforementioned improvements in cc5328a and https://github.com/buildpacks/docs/pull/571